swagger.json

```diff
@@ -5893,6 +5907,15 @@
                     "example": "111-222-3344"
                 }
             }
+        },
+        "ContactIds": {
+            "type": "array",
+            "description": "A list of IDs for contacts who can be notified about alerts",
```

Comment:
this description ties ContactIds exclusively to alerts, but in fact it's just a list of Ids and the type could be reused elsewhere.

Reply:
@silverspace requested in the spec for `Contacts` API that we describe contacts as `Information about a notification contact for alerts.`

Reply:
That makes sense in the context of the `Contact` object but this particular object is just a list of IDs. If we want to bring alerts into it can we explicitly tie it back to addresses also? this just looks wrong in cases where someone is just trying to create an address and is like, "hm, what is this list of contacts ids for?".

Reply:
We could update this description when we support more operations on address based contacts? I'm fine with whatever you guys decide for this description (and I'm also OOO this week, so won't be following this thread).

Reply:
I'm ok with rewording this so that it's not tied to alerts!

Reply:
@EmmaOnThursday updated the text here to be more general.